### PR TITLE
faq: cont'd #40

### PR DESCRIPTION
**Pull request by owanikin**
Implements issue #40

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ We have tied together all our datasets uploads here: [https://market.oceanprotoc
 
 You can work with existing ones! To get started, you might want to go with mock ones of course.
 
+## Can I submit a no-code solution, such as an entire static MVP like a really good design?
+
+Yes, absolutely. It just have to be really good!
```
Comment (epowell101) [inline docs/faq.md:43]: "It just has to be really good." as opposed to be it HAVE to be really good

**Review by epowell101 — CHANGES_REQUESTED** [review]
See minor comment HAVE vs. HAS

**Review by sarob — APPROVED** [review]
Looks good.

**Review by sarob — APPROVED** [review]
accepted

**Pull request merged**